### DISCUSSION TITLE
Multi-bit LEDs

### DISCUSSION THREAD

**Discussion by L1M1T0FM1NDS** [logisim-evolution/logisim-evolution#1918] [Ideas]
I've been thinking about tackling making a graphics card, and I'm trying to design it with more colours, but the simulation doesn't allow for that. I could design it without using multi-bit, but I'm having a hard time understanding what to do without it.
This might be a stupid request, but multi-bit LEDs would solve this problem, and would be an amazing addition.

**Comment by maehne** [comment]
Please, have a look at the RGB LED and the RGB Video components in the Input/Output library. They might already fulfill your requirements -- at least from what I understand from your request. For the moment, I will mark this as "won't fix", as it's already covered. Feel free to reopen, to discuss your request further and provide a more precise specification.
